tasks/need_fixing_linters/need_fixing_linters.py

```diff
@@ -0,0 +1,58 @@
+"""
+Invoke tasks to fix the linter
+"""
+
+from invoke import task
+from invoke.exceptions import Exit
+from ..libs.pipeline_notifications import GITHUB_SLACK_MAP
+from .golangci_lint_parser import filter_lints, display_result, merge_results
+
+FIRST_COMMIT_HASH = "52a313fe7f5e8e16d487bc5dc770038bc234608b"
+# See https://go.dev/doc/install/source#environment for all available combinations of GOOS x GOARCH.
+CI_TESTED_OS_AND_ARCH = {"linux": ["amd64", "arm64"], "windows": ["amd64"], "darwin": ["amd64", "arm64"]}
+
+def check_if_team_exists(team: str):
+    """
+    Check if an input team exists in the GITHUB_SLACK_MAP. Exits the code if it doesn't.
+    """
+    if team:
+        team = team.lower()
+        if not team in GITHUB_SLACK_MAP:
+            raise Exit(f"=> Team '{team}' does not exist.\n=> Your team should be in {[t for t in GITHUB_SLACK_MAP]}", code=2)
+    else:
+        print("[WARNING] No team entered. Displaying linters errors for all teams.\n")
+
+@task
+def need_fixing_linters(ctx, filter_team: str=None, from_commit_hash: str=FIRST_COMMIT_HASH, filter_linters: str="revive", show_output=False, tested_os_and_arch=CI_TESTED_OS_AND_ARCH):
```

Comment:
```suggestion
def show_linter_issues(ctx, filter_team: str=None, from_commit_hash: str=FIRST_COMMIT_HASH, filter_linters: str="revive", show_output=False, tested_os_and_arch=CI_TESTED_OS_AND_ARCH):
```
This name would reflect more the role of this task, wdyt?

Reply:
Agreed ! I added a `s` to linter since we can apply several linters per run. Fixed with [469f61e](https://github.com/DataDog/datadog-agent/pull/19015/commits/469f61e8db1d94c2e1f967a96c277ec0a454d041).